Jenkinsfile

```diff
@@ -0,0 +1,16 @@
+pipeline {
+    agent { docker { image 'ruby:2.4.4' } }
+    stages {
+        stage('rubocop') {
+            steps {
+                sh 'gem install rubocop'
+                sh 'src/bosh_azure_cpi/bin/rubocop_check'
+            }
+        }
+        stage('unit-test') {
+            steps {
+                sh 'src/bosh_azure_cpi/bin/unit-test'
```

Comment:
Why not use `test-unit` directly?

Reply:
ok, will merge these to one using the args.